### DISCUSSION TITLE
can prest compare two fields?

### DISCUSSION THREAD

**Discussion by LanHao0** [prest/prest#708] [API Server] [answered]
for now, I know how to use inner join, but how to compare two fields?

what I want to query :
```
SELECT * FROM public.a
inner join b on a.uuid=b.uuid
where b.i_count<b.j_count
```

**Comment by arxdsilva** [comment, marked as answer]
Hi @LanHao0 you could use our custom queries, see more at: https://docs.prestd.com/prestd/docs/api-reference/queries/

**Reply by LanHao0** [reply]
Thanks for answering! and yes, I saw the custom queries are handy! 
I'm also hoping prest can become simpler on comparing two fields in the future! (like [those](https://docs.prestd.com/prestd/api-reference/parameters/) !🔥🔥🔥)
I really like how simple and convenient prest is!

**Reply by arxdsilva** [reply]
No problem @LanHao0 ! It's great to see so many people using and liking our project!
